File: tests/fixtures/sample.java
import java.util.ArrayList;

class UserService {
    UserService() {}

    void greet(String user) {
        System.out.println(user);
    }

    static UserService create() {
        ArrayList<String> items = new ArrayList<>();
        items.add("hi");
        return new UserService();
    }
}

File: tests/fixtures/sample.cs
using System;
using System.Collections.Generic;

namespace Demo {
  public class UserService {
    public UserService() {}

    public void Greet(string user) {
      Console.WriteLine(user);
    }

    public static UserService Create() {
      var factory = (string name) => new UserService();
      var list = new List<string>();
      list.Add("x");
      return factory("alice");
    }
  }
}
